Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -32,8 +33,57 @@ using namespace o2::aod;
 using namespace o2::analysis;
 using namespace o2::analysis::hf_cuts_single_track;
 
+/// Event selection
+struct HfProduceSelCollisions {
+
+  Produces<aod::HFSelCollision> rowSelectedCollision;
+
+  Configurable<bool> b_dovalplots{"b_dovalplots", true, "fill histograms"};
+  Configurable<std::string> s_trigger_class{"trigger_class", "kINT7", "trigger class"};
+  int trigger_class = std::distance(aliasLabels, std::find(aliasLabels, aliasLabels + kNaliases, s_trigger_class.value.data()));
+
+  HistogramRegistry registry{
+    "registry",
+    {{"h_events", "Events;;entries", {HistType::kTH1F, {{3, 0.5, 3.5}}}}}};
+
+  void init(InitContext const&)
+  {
+    std::string labels[3] = {"processed collisions", "selected collisions", "rej. trigger class"};
+    for (int iBin = 0; iBin < 3; iBin++) {
+      registry.get<TH1>(HIST("h_events"))->GetXaxis()->SetBinLabel(iBin + 1, labels[iBin].data());
+    }
+  }
+
+  // event selection
+  void process(soa::Join<aod::Collisions, aod::EvSels>::iterator const& collision)
+  {
+    int status_collision = 0;
```

Comment:
Is `status_collision` supposed to store other info than rejection reasons?

Reply:
No, only rejection reasons I would say